lib/fastlane/plugin/wpmreleasetoolkit/helper/github_helper.rb

```diff
@@ -128,6 +139,28 @@ def self.download_file_from_tag(repository:, tag:, file_path:, download_folder:)
 
         download_path
       end
+
+      # Creates (or updates an existing) GitHub PR Comment
+      def self.comment_on_pr(project_slug:, pr_number:, body:, reuse_identifier: SecureRandom.uuid)
+        client = github_client
+        comments = client.issue_comments(project_slug, pr_number)
+
+        complete_reuse_identifier = "<!-- REUSE_ID: #{reuse_identifier} -->"
```

Comment:
💄  nit: I'd name this `reuse_marker` or something like this instead.

`complete_…` reminds me of "completion", like if we were trying to do auto-completion of the reuse ID… while this is more like a "marker comment" here.

Reply:
Makes sense – addressed in 76a7626cc548ad580390ea5687fef6027ba842f8